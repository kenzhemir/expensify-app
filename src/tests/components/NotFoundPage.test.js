import { shallow } from "enzyme";
import React from "react";

import NotFoundPage from "../../components/NotFoundPage";

test("should render expense list with expenses", () => {
	const wrapper = shallow(<NotFoundPage />);
	expect(wrapper).toMatchSnapshot();
});
