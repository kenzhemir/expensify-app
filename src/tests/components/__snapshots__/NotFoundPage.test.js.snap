// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list with expenses 1`] = `
<div>
  404 - 
  <Link
    to="/"
  >
    Home
  </Link>
</div>
`;
